### ISSUE TITLE
eml2eal() should import size, md5checksum, number rows, and download URLs

### ISSUE THREAD

**Issue by kzollove** [EDIorg/EMLassemblyline#110]
This would enable best possible transfer from ezEML to EAL with minimal data loss.

My particular situation is that I'm trying to migrate metadata created in ezEML with 72 tables to EAL. If the above parameters were passed to make_eml(), there would be no need to ever physically download the data tables (that are currently in ezEML and have static download links)

Download example ezEML data package [here](https://tinyurl.com/y2p3nvmk)

Is this too specific of a use case? Could play nicely with #107

**Comment by clnsmth** [comment]
Nice @kzollove. Yes, #107 should take care of this.

**Comment by clnsmth** [comment]
Yeah @kzollove, quote characters are a bit challenging here, especially in and out of a spreadsheet editor. The current [template_physical()](https://github.com/EDIorg/EMLassemblyline/blob/offline/R/template_physical.R) plays nicely with make_eml() but is not optimal (see the docs for `template_physical()`). Maybe you can find a better solution?

Ultimately physical.txt should return unscathed from the round trip of `template_physical()` > `make_eml()` > `eml2eal()`.
